pkg/build/docker_go.go

```diff
@@ -94,6 +94,10 @@ type DockerGoBuilderConfig struct {
 	// cached image.
 	EnableGoBuildCache bool `toml:"enable_go_build_cache"`
 
+	// Cgo enables the creation of Go packages that call C code. By default it is disabled.
```

Comment:
Leaving a note to ourselves, because I know we're probably going to trip on this one again.

```suggestion
	// Cgo enables the creation of Go packages that call C code. By default it is disabled.
	// Enabling CGO also enables dynamic linking. Disabling CGO (default) produces statically
	// linked binaries.
	//
	// If you ever see errors like the following, you are probably better off
	// disabling CGO (and therefore enabling static linking).
	//
	//   /testplan: error while loading shared libraries: libdl.so.2: cannot open shared object file: No such file or directory
```